mongo/mongoDbPrepare.js

```diff
@@ -61,7 +61,7 @@
 	db.transactionStatuses.createIndex({ 'status.hash': 1 }, { unique: true });
 	db.transactionStatuses.createIndex({ 'status.deadline': -1 });
 
-	db.adminCommand( { setFeatureCompatibilityVersion: '6.0' } )
+	db.adminCommand( { setFeatureCompatibilityVersion: '7.0' } )
```

Comment:
In another project, you upgrade to mongo:8.
Should it set it to 8 as well?

Reply:
No, for a couple of reasons
1. Mainnet is currently using MongoDB 6, and MongoDB only allows upgrades to the next version.
2. We have not tested MongoDB 8 yet. The other PR is to upgrade tests to start testing MongoDB 8.